admin/publish.md

```diff
@@ -10,7 +10,7 @@ Get access from the Docusaurus npm admins (@yangshun/@JoelMarcey).
 
 You need publish access to **the main Docusaurus repository** (not a fork).
 
-## npm
+## NPM
```

Comment:
Revert changes in this file?

Reply:
oh shit this is totally unwanted changes, thanks 👍